kale/nbparser/parser.py

```diff
@@ -66,14 +67,18 @@ def parse_metadata(metadata):
         # name of the future Pipeline step
         # TODO: Deprecate `block` in future release
         if tag_name in ["block", "step"] and value:
+            if tag_name == "block":
+                warnings.warn("`block` tag will be deprecated in a future "
+                              "version, use `step` tag instead",
+                              DeprecationWarning)
```

Comment:
@elikatsis  I am phrasing this generically as `future version` and not specifically like `v.0.5` because we don't really have a proper release process or feature roadmap. I think it's fine to leave it like this, wdyt?

Reply:
Yes, I totally agree.

nit (based on styling guidelines):
```python
warnings.warn("`block` tag will be deprecated in a future"
              " version, use `step` tag instead",
              DeprecationWarning)
```
We tend to put the space character to the start of the following line.